.idea/runConfigurations/Generate_FIR_Checker_Boilerplate.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
This file may be removed now :)

Reply:
Oops! I probably screw it up when amending my commit :(